inventory/pkg/client/client.go

```diff
@@ -576,6 +576,36 @@ func NewTenantAwareInventoryClient(
 	return cl, nil
 }
 
+// Set up the heartbeat ticker to keep the client connection alive.
+func (client *inventoryClient) heartbeat(clientUUID string) error {
+	const (
+		backoffRetries    = 3
+		backoffInterval   = time.Second * 1
+		heartbeatInterval = time.Second * 30
+	)
```

Comment:
let's make these param configurable. At least the retry and the heartbeat interval.